pydocusign/client.py

```diff
@@ -47,6 +48,8 @@ def __init__(self,
             self.account_url = '{root}/accounts/{account}'.format(
                 root=self.root_url,
                 account=self.account_id)
+        #: Requests timeout
```

Comment:
Here you can add details about the "timeout" meaning, i.e. you can tell it is used as timeout for HTTP requests to DocuSign API.